source/camera/advanced/capture_hdr_loop.py

```diff
@@ -19,7 +19,10 @@ def _main():
     camera = app.connect_camera()
 
     for i in range(1, 4):
-        settings_file = Path() / get_sample_data_path() / f"Settings/Zivid One/Settings0{i :01d}.yml"
+        camera_model = camera.info.model_name
+        camera_path = "Settings/" + camera_model[0:9] + f"/Settings0{i :01d}.yml"
+        settings_file = Path() / get_sample_data_path() / camera_path
```

Comment:
In C++ and C# these two are one line. I suggest to make them one line here as well.

https://github.com/zivid/zivid-cpp-samples/blob/d092e7bfe41aec7cd7a14281e28a3b04f9c4a383/source/Camera/Basic/CaptureWithSettingsFromYML/CaptureWithSettingsFromYML.cpp#L20-L23

https://github.com/zivid/zivid-csharp-samples/blob/ff844b4307451c61fba6ca92b279a0d9ceb0defc/source/Camera/Basic/CaptureWithSettingsFromYML/CaptureWithSettingsFromYML.cs#L20-L24

Reply:
I tried, it gets mad about how the path is made and throws errors.

Reply:
So it did not work to make them the exact same.

Reply:
Have you tried this:

`Path() / get_sample_data_path() / camera_path / Path("Settings/" + camera_model[0:9] + f"/Settings0{i :01d}.yml")`

Reply:
Fixed